synchronization/src/androidTestCyface/java/de/cyface/synchronization/CyfaceAuthenticatorTest.java

```diff
@@ -14,33 +13,29 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import android.accounts.Account;
-import android.accounts.AccountManager;
-import android.accounts.AccountManagerCallback;
-import android.accounts.AccountManagerFuture;
-import android.accounts.AuthenticatorException;
-import android.accounts.OperationCanceledException;
+import android.accounts.*;
 import android.content.Context;
 import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.preference.PreferenceManager;
-import androidx.test.platform.app.InstrumentationRegistry;
+import android.util.Log;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
 import androidx.test.filters.FlakyTest;
 import androidx.test.filters.LargeTest;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-import android.util.Log;
+import androidx.test.platform.app.InstrumentationRegistry;
 
 /**
  *
  * @author Klemens Muthmann
  * @author Armin Schnabel
- * @version 1.1.0
+ * @version 1.1.2
  * @since 2.0.0
  */
 
 @RunWith(AndroidJUnit4.class)
 @LargeTest
-@FlakyTest
+@FlakyTest // Flaky means (because of build.gradle) that this test is not executed in the Mock flavour (because it
```

Comment:
The way it is done now we could probably write our own annotation and base the exclusion on that annotation. That way we could use a more intuitive name.

Reply:
Interesting idea. Put it into a task if you fancy so ;-)